src/components/TextInput/BaseTextInput.js

```diff
@@ -53,6 +53,9 @@ class BaseTextInput extends Component {
         }
 
         this.value = this.props.value;
+        if (this.props.value === '') {
```

Comment:
It looks good but can you leave a comment before the if condition explaining why we need to do this?

Reply:
I have add the comment